src/Utils/NotificationConsoleUtils.ts

```diff
@@ -1,6 +1,10 @@
 import * as _ from "underscore";
 import { ConsoleDataType } from "../Explorer/Menus/NotificationConsole/NotificationConsoleComponent";
-
+declare global {

```

Comment:
This should not be added. Why is it necessary?

Reply:
I have remove all of the files related to Explorer